packages/squiggle-lang/src/rescript/Distributions/SampleSetDist/KdeLibrary.js

```diff
@@ -15,8 +15,18 @@ const samplesToContinuousPdf = (
   if (_.isFinite(max)) {
     _samples = _.filter(_samples, (r) => r < max);
   }
+
+  // The pdf that's created from this function is not a pdf but a pmf. y values
+  // being probability mass and not density.
+  // This is awkward, because our code assumes later that y is a density
   let pdf = pdfast.create(_samples, { size, width });
-  return { xs: pdf.map((r) => r.x), ys: pdf.map((r) => r.y) };
+
+  // To convert this to a density, we need to find the step size. This is kept
+  // constant for all y values
+  let stepSize = pdf[1].x - pdf[0].x;
+
+  // We then adjust the y values to density
+  return { xs: pdf.map((r) => r.x), ys: pdf.map((r) => r.y / stepSize) };
```

Comment:
this line does two things at one time (map values to arrays, then map y values to account for step size). It could be good to separate those into different lines, but it's not a big deal.